adaptive_navigation/example/lib/default_scaffold.dart

```diff
@@ -9,6 +13,7 @@ class DefaultScaffoldDemo extends StatefulWidget {
 class _DefaultScaffoldDemoState extends State<DefaultScaffoldDemo> {
   int _destinationCount = 5;
   bool _fabInRail = false;
+  bool _overflowDestinationAddedToMenu = true;
```

Comment:
Rename to `_includeBaseDestinationInMenu` to match the rail

Reply:
Done